backend/clubs/views.py

```diff
@@ -2617,6 +2617,7 @@ class QuestionAnswerViewSet(viewsets.ModelViewSet):
 
     serializer_class = QuestionAnswerSerializer
     permission_classes = [QuestionAnswerPermission | IsSuperuser]
+    lookup_field = "id"
```

Comment:
In general,`lookup_field` defaults to `id`, so this is not strictly necessary. Feel free to remove it.